File: 351001/Strizhak/rest_noSql/Discussion/Entities/Reaction.cs
using Cassandra.Mapping.Attributes;
namespace Discussion.Entities
{
    [Table("tbl_reaction")]
    public class Reaction
    {
        [PartitionKey]
        [Column("topic_id")]
        public long TopicId { get; set; }

        [ClusteringKey(0)]
        [Column("id")]
        public long Id { get; set; }

        [Column("content")]
        public string Content { get; set; }
    }
}

File: 351002/Stepanenko/lab4/discussion/src/main/java/by/bsuir/distcomp/discussion/domain/Reaction.java
package by.bsuir.distcomp.discussion.domain;

import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Table("tbl_reaction")
public class Reaction {

    @PrimaryKey
    private ReactionKey key;

    @Column("content")
    private String content;

    public ReactionKey getKey() {
        return key;
    }

    public void setKey(ReactionKey key) {
        this.key = key;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}
